### DISCUSSION TITLE
After Changing my username only the readme files are coming in my commits section ,why other main commits are not coming??

### DISCUSSION THREAD

**Discussion by farhanpavel** [community/community#72970] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://github.com/farhanpavel/Cgpa-result-processing/commits/main
Here Only My Read Me Part Is Coming As Commit But Why Other Commits are not counting. I Recently Change My GitHub Username.
https://github.com/farhanpavel/Cgpa-result-processing/commits?author=FarhanPavel

**Comment by b8zeek** [comment]
If you changed your GitHub username, the author information on past commits aren't automatically updated. So, I guess, your past commits may be associated with your old username, and they won't show up under your new username.

I guess you'll have to rebase... Download the repository, `git rebase` to start the session and edit the commits with the wrong author information with `git commit --amend --author="Your Name <your-email@address.com>".

Be careful, when doing this you can cause the problems for others if someone cloned or forked the repo.

Good luck.

🍀
